core/tests/test_utils.py

```diff
@@ -194,11 +228,25 @@ def _restore_stashed_user_env(self):
         self.stashed_user_env = None  # pylint: disable=attribute-defined-outside-init
 
     def login(self, email, is_super_admin=False):
+        """Gets User id from given email and checks if user is super admin.
```

Comment:
No check is made here. This simply sets the environment variables to simulate a login.